examples/depth-filters/readme.md

```diff
@@ -0,0 +1,233 @@
+# rs-align Sample
```

Comment:
This entire file is a copy&paste of `rs-align` example.

Reply:
Will be removed